components/Chip.js

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
```

Comment:
yes makes sense, not to introduce another word.
It comes from the lib that we introduced for buttons, but i figured out making our own chip/label/badge is easier for us (since it has no action etc.). https://reactnativeelements.com/docs/chip

anyways i am going with `labels` (since this app is closely related to aplus).